app/views/home/index.html.haml

```diff
@@ -4,6 +4,11 @@
   html5up.net | @n33co
   Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
 %html{lang: 'en'}
+  - unless ENV["DISABLE_SANDBOX_WARNING"]
+    %div{class: "sandbox-banner"}
+      %span
+        This sandbox site is for testing and training purposes only. To view and make live requests,
+        %a{href: "https://cap.18f.gov"} go to the live site.

```

Comment:
Mind making a shared partial for this, and using it on this page and in the partial below?

http://guides.rubyonrails.org/layouts_and_rendering.html#using-partials